observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,103 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-9_{context}"]
+== Network Observability Operator 1.9
+The following advisory is available for the Network Observability Operator 1.9:
+
+* link:https://access.redhat.com/errata/RHSA-2025:10020 [Network Observability Operator 1.9]
+
+=== New features and enhancements
+
+[id="user-defined-networks-with-network-observability_{context}"]
+==== User-defined networks with Network Observability
+With this release, the user-defined networks (UDN) feature is generally available with Network Observability. When the `UDNMapping` feature is enabled in Network Observability, the *Traffic* flow table has a `UDN labels` column. You can filter on *Source Network Name* and *Destination Network Name*.
+
+[id="filter-flowlogs-at-ingestion_{context}"]
+==== Filter flowlogs at ingestion
+With this release, you can create filters to reduce the number of generated network flows and the resource usage of Network Observability components.
+
+You can configure the following filters:
+
+* eBPF Agent filters
+* Flowlogs-pipeline filters
+
+[id="ipsec-support_{context}"]
+==== IPsec support
+With this update, when IPsec is enabled on {product-title}:
+
+* A new column named *IPsec Status* is displayed in the Network Observability *Traffic* flows view to show whether a flow was successfully IPsec-encrypted or if there was an error during encryption/decryption.
+
+* A new dashboard showing the percentage of encrypted traffic is generated.
+
+[id="network-observability-cli-1-9_{context}"]
+==== Network Observability CLI
+New filtering options available with packets, flows, and metrics capture:
+
+* Track IPsec using `--enable_ipsec`
```

Comment:
I wonder how this would look if you reformatted to something like

```suggestion
`--enable_ipsec`:: Tracks IPsec metrics 
```

Technically this is how you'd do definition lists (https://www.ibm.com/docs/en/ibm-style?topic=format-lists), but I don't what you've written is 100% a _definition list_. Just a thought.